knowledge-base/grid-datasourcerequest-on-server.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Use Telerik DataSourceRequest and DataSourceResult on the server
+description: How to use Telerik DataSourceRequest and DataSourceResult on the server.
+type: how-to
+page_title: Telerik DataSourceRequest and DataSourceResult
+slug: grid-kb-datasourcerequest-on-server
+position: 
+tags: 
+res_type: kb
+---
+
+
+## Description
+
+How to send the Grid state to the server and retrieve data? How to use Telerik `DataSourceRequest` and `DataSourceResult` on the server?
```

Comment:
Add a sentence about the benefits - like optimization of queries and amount of loaded data, keywords like server paging, server filtering

Reply:
Added as relevant questions in the description.